composer.json

```diff
@@ -39,7 +39,7 @@
 		}
 	],
 	"require": {
-		"php": ">=5.3.3",
+		"php": ">=5.4",
 		"composer/installers": "~1.0"
```

Comment:
@michaelcullum Sorry but I don't get the point of this. Doing this makes composer try to install phpbb into the extension's vendor folder, which is not good (hence one of the reasons for using the soft require). It also fails running composer for an extension, currently because:
>   Problem 1
    - Installation request for phpbb/phpbb >=3.2.0@dev -> satisfiable by phpbb/phpbb[dev-master].
    - phpbb/phpbb dev-master requires s9e/text-formatter dev-release/php5.3 -> no matching package found.